internal/bookmark.go

```diff
@@ -90,13 +91,13 @@ func setBookmark(ctx context.Context, doc *enigma.Doc, bookmarkID string, raw js
 		return err
 	}
 	if bookmark.Handle != 0 {
-		LogVerbose("Updating bookmark " + bookmarkID)
+		log.Debugln("Updating bookmark " + bookmarkID)
```

Comment:
I don't think this could count as debug🤓

Reply:
I am tempted to agree, but, the verbose level is equivalent with the new debug. This might not be ideal, since I agree that it's not a debugging message, but the other alternative would be to put it as info instead, meaning it would be default logging.

Also, to be consistent, we should then change this to Info for all similar operations e.g. objects, measures etc... which would potentially flood the terminal. (If by default enabled.)

Maybe change the log levels to: DEBUG > INFO2? > INFO instead of TRACE > DEBUG >INFO as it is now. (Trace isn't even used at this moment so it wouldn't be that strange.)
Thoughts @wennmo @gabbaxx ?

Reply:
I agree that we can skip `TRACE`, but otherwise I think we should stick to the logging conventions of Qlik. Assuming that `INFO2` was a typo? 😉